testing-modules/selenium/pom.xml

```diff
@@ -56,8 +56,8 @@
     </build>
 
     <properties>
-        <testng.version>6.10</testng.version>
-        <selenium-java.version>4.8.3</selenium-java.version>
+        <testng.version>7.9.0</testng.version>
```

Comment:
Have we checked that increasing these dependencies doesn't break the other code examples?

Reply:
Reran all the tests, most of the tests ran successfully. The following observations were found : 
1. The following tests inside the package >> “stale” were throwing error  - “org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Response code 500. Message: session not created: This version of ChromeDriver only supports Chrome version 114.” Upgraded the WebDriverManager dependency version to latest version(5.7.0) and it worked fine:
StaleElementReferenceLiveTest
RobustWebElementLiveTest

2. The following test in SeleniumPageObjectLiveTest class fails as it couldn’t locate the WebElement with locator - “.page-title”
givenHomePage_whenNavigate_thenShouldBeInStartHere()

3. The following test in SeleniumPageObjectLiveTest class fails as it couldn’t locate the WebElement with locator - “.page-header”
givenAboutPage_whenNavigate_thenTitleMatch()

The following test in TakeScreenShotSeleniumLiveTest class fails as it couldn’t locate the WebElement with locator - “#hplogo”
whenGoogleIsLoaded_thenCaptureLogo()